### PR TITLE
[ODS-4297 , ODS-4287] Cleanup .net.cs files and pre-processor directives , Drop net48 support in class libraries

### PR DESCRIPTION
**Pull request by BhanuprakashThunga**

### CODE REVIEW

File: Application/EdFi.Ods.Tests/EdFi.Ods.Tests.csproj [hunk 1]
```diff
@@ -122,4 +119,44 @@
...
+    <Folder Include="EdFi.Ods.Pipelines\Steps\" />
+    <Folder Include="EdFi.Ods.Repositories.NHibernate\Composites\" />
+    <Folder Include="EdFi.Ods.Sandbox\Security\" />
+    <Folder Include="EdFi.Ods.Security\Utilities\CastleWindsor\" />
+    <Folder Include="EdFi.Ods.WebApi.Common\" />
+    <Folder Include="EdFi.Ods.WebApi\Controllers\" />
+    <Folder Include="EdFi.Ods.WebApi\NHibernate\Architecture\" />
+    <Folder Include="EdFi.Security\Authorization\" />
+    <Folder Include="EdFi.Security\Claims\" />
+  </ItemGroup>
```
Comment (blmeyers) [inline Application/EdFi.Ods.Tests/EdFi.Ods.Tests.csproj:161]:
```suggestion
```

File: Application/EdFi.Ods.Tests/_Extensions/ExceptionAssertionExtensions.cs [hunk 2]
```diff
@@ -32,13 +31,12 @@ public static void MessageShouldContain(this Exception exception, string expecte
         {
             Assert.That(exception?.Message, Does.Contain(expectedSubstring),
                 $@"Exception message does not contain the expected text.
-Actual Exception: {exception}");
+                Actual Exception: {exception}");
```
Comment (blmeyers) [inline Application/EdFi.Ods.Tests/_Extensions/ExceptionAssertionExtensions.cs:34]: Revert extra formatting changes please.

File: tests/EdFi.Common.UnitTests/EdFi.Common.UnitTests.csproj [hunk 3]
```diff
@@ -53,6 +48,11 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Configuration\" />
+    <Folder Include="InversionOfControl\" />
+  </ItemGroup>
```
Comment (blmeyers) [inline tests/EdFi.Common.UnitTests/EdFi.Common.UnitTests.csproj:55]:
```suggestion
```

**Review by blmeyers — CHANGES_REQUESTED** [review]
Addressed a few areas in comments, but in general:

1. Don't explicitly include folders in the csprojs, particularly when they're folders that are being deleted.
2. Make sure we're cleaning up all the code we're deleting, not just excluding it.
3. Revisit all the csproj files.  There are still cases of framework checks sticking around, both for standard and for net48.  After this PR there should be no more conditional includes in the csproj files.
4. There are still 43 files with #if conditionals.  Please make sure we're removing the conditional if its for NETSTANDARD or CORE, and that we're deleting the file if its for net48.
5. Take a pass through all the projects and make sure we've cleaned up all the .config files except the web.configs in the web projects for IIS support.  There should not be any more app configs now. Also clean up the hack in the csproj for copying the app.config for some of the test projects.

**Review by jamessmckay — APPROVED** [review]

**Review by blmeyers — APPROVED** [review]

**Pull request merged**